hack/scripts/create_scan_config.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# This script has the following arguments:
+# - filename of file to be created (mandatory)
+# - release tag (mandatory)
+# ./create_scan_config image temp_scan_config.yaml tag
+# - use when bumping the config on the main branch
+
+FILENAME=${1}
+TAG=${2}
```

Comment:
Please let's check if these input exist or not, and fail if they don't.

Reply:
updated the script. `set -o nounset` will complain if any arg or var is not defined.